neo4j/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG - neo4j
 
+## 2.2.0 / 2022-03-28
```

Comment:
```suggestion
## 3.0.0 / 2022-03-28
```
This should be a major version update since there is a `[Changed]` changelog entry.